tutorials/article.txt

```diff
@@ -0,0 +1 @@
w to Ellen DeGeneres in which she said that she was used to being bullied about her weight . Recently Kelly Clarkson gave an interview to American chat show host Ellen DeGeneres in which she discussed how she had been facing criticism for her weight for years. She said: 'I’m such a creative person that I yo yo. So sometimes I’m more fit and I get into kickboxing hardcore. And then sometimes I don’t and I’m like.. I’d rather have wine.' 'We are who we are. Whatever size,' Kelly told Ellen, prompting applause from the studio audience. 'You know, if it’s you, you’re just like whatever,' she said. 'I think what hurts my feelings..... is that I’ll have a meet and greet after the show and a girl who is, like, bigger than me will be in the meet and greet and be like, 'wow, if they think you’re big, I must be so fat to them.' In 2012 Girls star and producer Lena Dunham hit back after she was criticised for wearing a pair of short shorts on the red carpet. In 2012 Lena Dunham recieved harsh criticism for this outfit. She said she didn't think a thinner woman would have been criticised in the same way . Lena told her critics: 'I am going to live to be 100, and I am going to show my thighs every day till I die.' She said at the time: 'Last week I wore something to an event...a big top and little shorts, and a bunch of [blog posts] came out that I had been out without pants [trousers],' 'I actually saw it...'Love it or hate it: The no-pants look.' My mom...thought it was so funny. 'My boyfriend was like, 'People seem to be worked up about you going out without pants.' But I didn't go out without pants! I had shorts on.' Lena continued: 'If Olivia Wilde had gone to a party in...little shorts, she might have been on a 'weird dressed list' or been told her outfit was cute. 'I don't think a girl with tiny thighs would have received such no-pants attention. I think what it really was...'Why did you all make us look at your thighs?' 'My response is, get used to it because I am going to live to be 100, and I am going to show my thighs every day till I die.' Tyra Banks' weight has gone up and down over the years and the model says that anyone who doesn't like the way she looks can kiss her fat a** . Former supermodel Tyra Banks similarly hit back after she was subject to mean remarks about her fuller figure. She said: 'To all of you that have something nasty to say about me, or other women that are built like me, women that sometimes or all the time look like this, women whose names you know, women whose names you don't, women who have been picked on, women whose husbands put them down, women at work or girls in school — I have one thing to say to you. Kiss my fat a**!' In 2014 when she was criticised for her appearance at the Golden Globes actress Gabourey Sidibe had the perfect response to her haters. She said: 'To the people making mean comments about my [Golden Globes] pics, I mos def cried about it on that private jet on the way to my dream job last night #jk.'
```

Comment:
lets move this to some kind of `/assets` dir

Reply:
Moved it into a path of `instructor/tutorials/assets/article.txt`